specs/binding-grpc-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/grpc/kafka/streams/kafka/produce/unary.rpc.error/client.rpt

```diff
@@ -0,0 +1,110 @@
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+connect "zilla://streams/kafka0"
+    option zilla:window 8192
+    option zilla:transmission "duplex"
+
+write zilla:begin.ext ${kafka:beginEx()
+                             .typeId(zilla:id("kafka"))
+                             .merged()
+                                 .capabilities("PRODUCE_ONLY")
+                                 .topic("requests")
+                                 .partition(-1, -2)
+                                 .ackMode("LEADER_ONLY")
+                                 .build()
+                             .build()}
+
+connected
+
+write zilla:data.ext ${kafka:dataEx()
+                                 .typeId(zilla:id("kafka"))
+                                 .merged()
+                                     .deferred(0)
+                                     .partition(-1, -1)
+                                     .key("test")
+                                     .header("zilla:identity", "test")
+                                     .header("zilla:service", "example.EchoService")
+                                     .header("zilla:method", "EchoUnary")
+                                     .header("zilla:reply-to", "responses")
+                                     .header("zilla:correlation-id", "59410e57-3e0f-4b61-9328-f645a7968ac8-479f2c3fb58bc3f04bbe15440a657670")
+                                     .build()
+                                 .build()}
+write ${grpc:protobuf()
+            .string(1, "Hello World")
+            .build()}
+write flush
+
+write zilla:data.ext ${kafka:dataEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .deferred(0)
+                                .partition(-1, -1)
+                                .key("test")
+                                .header("zilla:identity", "test")
+                                .header("zilla:service", "example.EchoService")
+                                .header("zilla:method", "EchoUnary")
+                                .header("zilla:reply-to", "responses")
+                                .header("zilla:correlation-id", "59410e57-3e0f-4b61-9328-f645a7968ac8-479f2c3fb58bc3f04bbe15440a657670")
+                                .build()
+                            .build()}
+
+write flush
+
+write close
+write notify SENT_ASYNC_REQUEST
+read closed
+
+connect await SENT_ASYNC_REQUEST
```

Comment:
Why it is called `ASYNC` as far as I know there is no such terminology in grpc as well

Reply:
Async over a pair of Kafka topics.